deploy/blobfuse2/config.yaml

```diff
@@ -0,0 +1,24 @@
+allow-other: true
+
+logging:
+  type: base
+  level: log_debug
+
+components:
+  - libfuse
+  - file_cache
+  - attr_cache
+  - azstorage
+
+libfuse:
+  attribute-expiration-sec: 120
+  entry-expiration-sec: 120
+  negative-entry-expiration-sec: 240
+
+file_cache:
+  path: /tmp/blobfuse/
```

Comment:
in blobfuse, we have `--tmp-path` mount option, do we have corresponding mount option in blobfuse2?

Reply:
let me remove this `path` parameter first.